src/mina-signer/src/sign-zkapp-command.ts

```diff
@@ -17,6 +17,7 @@ import {
   signFieldElement,
   verifyFieldElement,
 } from './signature.js';
+import { Pickles } from '../../snarky.js';
```

Comment:
this makes the JSOO output a dependency of mina-signer, which we don't want (it's supposed to be small)

To fix this, I recommend to autogenerate the verification key hash as a constant in `snarky_js_constants.ml`

Reply:
Gotcha, will do!